index.html

```diff
@@ -8,50 +8,266 @@
-            </p>
-            <p class="contact">LET’S CONNECT</p>
+            <div>
+                <h1 class="main-header">I’m Ibrahim <br/> Glad to see you!</h1>
+                <p class="main-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="main-items">
+                <p class="contact">LET’S CONNECT</p>
+                <ul class="icons-list list-style ">
+                    <li class="icons">
+                        <a href="https://twitter.com/ibr_ahmat" target="”_blank”" rel="noreferrer noopener"><img src="./assets/twitter.svg" alt="Twitter icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/" target="”_blank”" rel="noreferrer noopener"><img src="./assets/linkedin.svg" alt="Linked-In icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="#" rel="noreferrer"><img src="./assets/medium.svg" alt="Medium icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="https://github.com/ibr5500" target="”_blank”" rel="noreferrer noopener"><img src="./assets/github.svg" alt="Github icon" /></a>
+                    </li>
+                    <li class="icons">
+                        <a href="# " rel="noreferrer"><img src="./assets/angellist.svg" alt="Angellist icon" /></a>
+                    </li>
+                </ul>
+            </div>
         </section>
-        <ul class="icons list-style ">
-            <li>
-                <a href="https://twitter.com/ibr_ahmat" target="_blank" rel="noreferrer noopener"><img src="./assets/twitter.svg" alt="Twitter icon" /></a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/" target="_blank" rel="noreferrer noopener"><img src="./assets/linkedin.svg" alt="Linked-In icon" /></a>
-            </li>
-            <li>
-                <a href="#" rel="noreferrer"><img src="./assets/medium.svg" alt="Medium icon" /></a>
-            </li>
-            <li>
-                <a href="https://github.com/ibr5500" target="_blank" rel="noreferrer noopener"><img src="./assets/github.svg" alt="Github icon" /></a>
-            </li>
-            <li>
-                <a href="# " rel="noreferrer"><img src="./assets/angellist.svg" alt="Angellist icon" /></a>
-            </li>
-        </ul>
     </main>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/avocode-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
```

Comment:
- [ ] There should be only one `<h1>` tag on a page for semantic and SEO reasons. Kindly replace `<h1>` tag other than the one used for your name in the first section with `<hx>` tags (h2. .... h6).